app/components/Channels/ChannelData.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+import { opacity } from 'styled-system'
+import { Box as BaseBox } from 'rebass'
+import { blockExplorer } from 'lib/utils'
+import { Bar, DataRow, Link } from 'components/UI'
+import { Truncate } from 'components/Util'
+import { CryptoValue } from 'containers/UI'
+import messages from './messages'
+
+const Box = styled(BaseBox)(opacity)
+
+const ChannelData = ({ channel, currencyName, networkInfo, viewMode, ...rest }) => {
+  const {
+    channel_point,
+    commit_fee,
+    fee_per_kw,
+    total_satoshis_sent,
+    total_satoshis_received,
+    csv_delay,
+    num_updates
+  } = channel
+  const fundingTxid = channel_point.split(':')[0]
```

Comment:
Would be a little more cleaner using destructuring 
`const [fundingTxid] = channel_point.split(':')`

Reply:
Updated.